Index: CIRTWebsiteDjango/cirtwebsite/entrypoint.sh
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>#!/bin/bash\r\nset -o pipefail\r\n\r\necho \"Checking if netcat is installed...\"\r\nwhich nc || echo \"Netcat is NOT installed!\"\r\n\r\necho \"Installing Netcat...\"\r\n/bin/bash\r\napt update && apt-get install -y netcat-traditional\r\n\r\necho \"Waiting for MySQL to start...\"\r\nwhile ! /usr/bin/nc -z db 3306; do\r\n  echo \"Waiting for MySQL...\"\r\n  sleep 1\r\ndone\r\necho \"MySQL is up!\"\r\n\r\necho \"Applying database migrations...\"\r\npython manage.py migrate\r\n\r\n#echo \"Collecting static files...\"\r\n#python manage.py collectstatic --noinput\r\n\r\necho \"Starting Gunicorn...\"\r\nexec gunicorn --bind 0.0.0.0:8000 --workers 3 cirtwebsite.wsgi:application
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/CIRTWebsiteDjango/cirtwebsite/entrypoint.sh b/CIRTWebsiteDjango/cirtwebsite/entrypoint.sh
--- a/CIRTWebsiteDjango/cirtwebsite/entrypoint.sh	(revision c1dfe6b94dbab4dc5869740f995f53fd8a51e9e5)
+++ b/CIRTWebsiteDjango/cirtwebsite/entrypoint.sh	(date 1742917929680)
@@ -1,5 +1,5 @@
 #!/bin/bash
-set -o pipefail
+set -eo pipefail
 
 echo "Checking if netcat is installed..."
 which nc || echo "Netcat is NOT installed!"
